app/models/rent.rb

```diff
@@ -6,7 +6,7 @@ class Rent < ApplicationRecord
   after_create :send_mail
 
   scope :between_from_dates, -> { where('DATE(?) BETWEEN init_date AND end_date', Time.current) }
-
+  scope :expired_date, -> { where(end_date: Time.current) }
```

Comment:
```suggestion
  scope :expired_date, -> { where(end_date: Time.zone.today) }
```
because `Time.current` is a Time not a Date